### ISSUE TITLE
Test functions against all Data types

### ISSUE THREAD

**Issue by domjtalbot** [domjtalbot/heritagebot#17]
Currently functions are being tested against specific data sets which pass the tests.
Additional tests need to be added for each data type as the properities vary.

The convert function is failing for some 'polygon' based datasets like Wrecks.

The datasets to test against are:
- [ ] BattleFields
- [ ] Building Preservation Notices
- [ ] Certificates of Immunity
- [ ] Listed Buildings
- [ ] Parks & Gardens
- [ ] Protected Wrecks
- [ ] Scheduled Monuments
- [ ] Word Heritage Sites

**Comment by domjtalbot** [comment]
Added new story to [standardise all datasets](https://github.com/domjtalbot/heritagebot/issues/24) to one defined schema to make this easier.
Shall resume this bug once #24 is complete.
